_rules/lists-follow-content-model-a73be2.md

```diff
@@ -0,0 +1,277 @@
+---
+id: a73be2
+name: List elements follow content model
+rule_type: atomic
+description: |
+  This rule checks that list elements have child nodes allowed by the content model
+accessibility_requirements:
+  wcag20:1.3.1: # Info and Relationships (A)
+    forConformance: true
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+input_aspects:
+  - Accessibility tree
+  - CSS Styling
+  - DOM Tree
+acknowledgments:
+  authors:
+    - Brian Bors
+    - Wilco Fiers
+  previous_authors:
+    - Jey Nandakumar
+---
+
+## Applicability
+
+This rule applies to any [HTML elements][] with a [hidden state][] of "false", where the [semantic role][] is the same as the [implicit role][], and for which one of the following is true:
+
+- **lists**: The element is a `ul`, `ol` or `menu`; or
+- **definition lists**: The element is a `dl`, or a `div` that is a [child][] in the [flat tree][] of a `dl`.
+
+## Expectation
+
+For each [child][] in the [flat tree][] of each test target, one of the following is true:
+
+- **implicit role**: the [child][] follows the [content model][] and has no [explicit role][]; or
+- **explicit role**: the [child][] has an [explicit role][] which is the [implicit ARIA semantics][] for an HTML element included in the [content model][] for the test target; or
+- **hidden**: the [child][] is not [included in the accessibility tree][].
```

Comment:
I'd prefer calling this "not exposed" or similar, since "hidden" also has visual meaning which we don't care about here.

Reply:
Don't much like the word "exposed", it's not the most pleasant word, nor is it particularly descriptive here. Would you be okay with "not included"?

Reply:
Perfectly fine.